src/main/java/edu/kis/powp/jobs2d/command/CommandParser.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+public interface CommandParser {
+	void fillListFromFile() throws FileNotFoundException;
+	List<DriverCommand> getCommandList();
```

Comment:
I suggest a string as an argument.
The result could be simply a DriverCommand (not a list) because we can use ICompoundCommand.